roles/redis/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Install Redis and PHP extension
```

Comment:
1. Do we want to only install redis and/or memcache when they are enabled in `wordpress_sites.yml`?
2. Do we want to uninstall redis and/or memcache when they are disabled in `wordpress_sites.yml`?
    - Note: This might be problematic if users using redis more than caching.

Reply:
https://github.com/roots/trellis/blob/2cb663d6396c4911f7cd19b13fc9cd50f3298959/server.yml#L31

role is only run based on conditional which could be used here?